code/SINGE.m

```diff
@@ -0,0 +1,42 @@
+function [ranked_edges, gene_influence] = SCINGE(gene_list,Data,outdir,num_replicates,param_list)
```

Comment:
Let's remove the old `SCINGE.m` if it is no longer needed.  How much of this file changed?  Just the name?